### PR TITLE
Zoom

### PR DESCRIPTION
**Pull request by Nielswps**
Zoom functionality

### CODE REVIEW

**Review by yrke — CHANGES_REQUESTED** [review]
Something seems to kill performance in this branch, it gets really slow moving locations around on the screen.

File: src/main/java/ecdar/presentations/Grid.java [hunk 1]
```diff
@@ -91,11 +91,10 @@ public void updateGrid(double scale, double width, double height) {
             horizontalLines.forEach(line -> getChildren().add(line));
         });
 
-        //Center the grid on the screen
-        double newHeight = (height / scale - (height / scale)) * 1.2;
-        double newWidth = (height / scale - (width / scale) / 1.1);
+        System.out.println("v: " + this.verticalLines.size() + " h: " + this.horizontalLines.size());
```
Comment (yrke) [inline src/main/java/ecdar/presentations/Grid.java]: I guess this should be removed, or only for debugging?

Comment (Nielswps) [reply src/main/java/ecdar/presentations/Grid.java]: Yes, sorry, ran a search through the entire project for "System.out", but I guess I missed the one. I will commit in a minute

Comment (Nielswps) [reply src/main/java/ecdar/presentations/Grid.java]: I have already removed the print line, but it seems that the current state of the PR does not correspond with the Nielswps:zoom branch. I will figure out why the PR is not up-to-date, turning it back into a draft and then re-open it when I have ensured that the PR reflects the branch changes... Sorry for the inconvenience

Comment (Nielswps) [reply src/main/java/ecdar/presentations/Grid.java]: I can see it was due to a filter, the updated version should already be available

**Review by yrke — APPROVED** [review]

**Pull request merged**